Cargo.toml

```diff
@@ -3,6 +3,7 @@ members = [
     "eth2/types",
     "eth2/utils/int_to_bytes",
     "eth2/utils/ssz_types",
+    "eth2/simulation",
```

Comment:
What is the purpose of the eth2 directory?

Reply:
eth2 -> holds eth2-related concepts (simulation, eth2 types, helper fns for eth2 stuff)
notion-server -> holds the http server logic
notion-cli -> holds the CLI for talking with the server